Kitodo/src/test/java/org/kitodo/production/services/data/DocketServiceIT.java

```diff
@@ -44,6 +46,7 @@ public static void prepareDatabase() throws Exception {
         MockDatabase.insertClients();
         MockDatabase.insertDockets();
         MockDatabase.setUpAwaitility();
+        SecurityTestUtils.addUserDataToSecurityContext(new User(), 1);
```

Comment:
Use insertRolesFull() instead of insertClients() and use real user here.

Reply:
I don't need the user, so i don't want to change insertion here